src/main/java/org/hoverla/bibernate/connectionpool/exception/ConfigurationException.java

```diff
@@ -0,0 +1,10 @@
+package org.hoverla.bibernate.connectionpool.exception;
```

Comment:
I think it's better to have an exception package. And inside have packages for specific exceptions, like connection pool, annotation evaluation, etc. This way we can have all the exceptions in one place.